spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/NativeEnvironmentRepository.java

```diff
@@ -49,13 +49,15 @@
  * @author Dave Syer
  * @author Roy Clarkson
  * @author Venil Noronha
+ * @author Daniel Lavoie
  */
 @ConfigurationProperties("spring.cloud.config.server.native")
-public class NativeEnvironmentRepository implements EnvironmentRepository, SearchPathLocator, Ordered {
+public class NativeEnvironmentRepository
+		implements EnvironmentRepository, SearchPathLocator, Ordered {
 
 	private static Log logger = LogFactory.getLog(NativeEnvironmentRepository.class);
-
-	private static final String DEFAULT_LABEL = "master";
+	
+	private String defaultLabel;
```

Comment:
I think this still needs to default to `master`. If a user creates a `NativeEnvironmentRepository` bean, they wouldn't know they might need to update this.

Reply:
`DEFAULT_LABEL` was never used. Exposed through `getDefaultLabel` but never used.

Reply:
right. `private String defaultLabel = "master"` is all I'm saying.

Reply:
In fact, the `profile` parameter will only be used if the users specify a custom location with `{label}` placeholder.

Updating to default to master.

Reply:
I guess if anyone relied on calling the public `getDefaultLabel()`, it's better to not break it. Fair enough.